Dockerfile

```diff
@@ -1,10 +1,6 @@
-FROM golang:1.11.10-alpine as builder
+FROM golang:1.13.10-alpine3.11 as builder
```

Comment:
hmm, i wouldn't have expected the trailing version number on `alpine`

Reply:
It isn't strictly necessary. It's to pin the version. You can also specify just `golang:1.13-alpine`, to leave point release from Go and version entirely from alpine, and it will just pick up latest of each.

I thought including full versions would be good for repeatability -- always using the same version of Go and alpine to build the docker image.